addon_service/authorized_storage_account/models.py

```diff
@@ -76,6 +76,12 @@ class AuthorizedStorageAccount(AddonsServiceBaseModel):
         related_name="authorized_storage_accounts",
     )
 
+    is_oauth1_ready = models.BooleanField(
+        "addon_service.OAuth2TokenMetadata",
+        null=True,
+        blank=True,
+    )
+
```

Comment:
I am not quite sure whether this boolean field is necessary.

Reply:
It is necessary as it allows to discern temporary Request token credentials from final credentials